contract/tests/test_contract.py

```diff
@@ -1593,3 +1593,27 @@ def test_analytic_account(self):
         # Case 4: two contract lines, each one with one analytic account
         new_contract_line.analytic_distribution = {self.analytic_account_2.id: 100}
         self.assertFalse(self.contract.group_id)
+
+    def test_analytic_distribution(self):
+        # Create an analytic distribution model for product 1
+        analytic_plan = self.env["account.analytic.plan"].create(
+            {
+                "name": "Plan",
+            }
+        )
```

Comment:
For avoiding black doing a lot of line, simply remove trailing commas:


```suggestion
        analytic_plan = self.env["account.analytic.plan"].create(
            {"name": "Plan"}
        )
```

Reply:
Done